hypershift-operator/controllers/manifests/manifests.go

```diff
@@ -62,3 +64,22 @@ func IngressDefaultIngressController() *operatorv1.IngressController {
 		},
 	}
 }
+
+func DummyRoute(namespace string) *routev1.Route {
+	return &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "dummy",
```

Comment:
can we give this a slightly more descriptive name, just in case someone encounters it.

something like `kubevirt-infra-temp-route`?